mercury-operator/src/main/java/com/redhat/mercury/operator/controller/ServiceDomainClusterController.java

```diff
@@ -90,7 +83,33 @@ public DeleteControl deleteResource(ServiceDomainCluster sdc, Context<ServiceDom
         return UpdateControl.updateStatusSubResource(sdc);
     }
 
-    private void updateStatusWithKafkaBrokerUrl(ServiceDomainCluster sdc, Context<ServiceDomainCluster> context, ServiceDomainClusterStatus status) {
+    private boolean isKafkaBrokerUrlInCluster(ServiceDomainCluster serviceDomainCluster) {
+        return serviceDomainCluster != null && serviceDomainCluster.getStatus() != null && serviceDomainCluster.getStatus().getKafkaBroker() != null;
+    }
+
+    private void createOrUpdateConfigMap(String sdcName, String kafkaBrokerUr) {
```

Comment:
I prefer that the bootstrap-servers goes as an environment variable in the image deployment, this way, if the server url is updated the deployments can be redeployed automatically.

Reply:
It is an environment variable, but it is also displayed in the service doman cluster status

Reply:
it is a configmap, that you mount as an env var. If the configmap changes, the deployment won't notice. It's better if it is directly an env var for the reasons I mentioned

Reply:
Do you mean I should change isKafkaBrokerUrlInCluster method to check it from the config map ?
Since the deployment reads it from the config map env var already